app/controllers/entries_controller.rb

```diff
@@ -159,7 +159,22 @@ def destroy
   end
 
   def files
-    blob = @entry.files.blobs.find_by!(filename: params[:filename])
+    # TODO: test this obviously insane behaviour
```

Comment:
i think i did this in order to support not-yet-important files as entries. but why did i do this in the normal entries controller as opposed to the static entries controller?